### ISSUE TITLE
meta-autoset:

### ISSUE THREAD

**Issue by ronc617** [marcelo-mason/PreciousStones#1117]
if the meta-autoset: is set to TRUE
the plugin works
if the meta-autoset: is set to false it will not regster that the field was placed

when a player crafts Diamond BLock it autosets the meta instead of leaving it as a diamond block 
untill the field is placed

**Comment by frizzbee30** [comment]
We had related issues to this some time ago, we don't use the meta autoset, but the dev was aware. Our fields work fine without it enabled via config. I have to be honest, it was so long ago i can't remember the issue/resolution

**Comment by frizzbee30** [comment]
Hi, unless the meta-autoset: is set to true, placing the blocks in survival mode fails to generate a field?

Thanks

 @marcelo-mason
Owner
marcelo-mason commented on May 29, 2016
You got 3 options:

Enable meta-autoset and have players mine the blocks as meta'd pstones
Disable meta-autoset and create the meta'd blocks via shops or other means and give them out to players. The plus side being they will still be able to mine the regular block as normal.
Remove the meta flags altogether to have them behave like default.
 @frizzbee30
  
frizzbee30 commented on May 29, 2016
Hi, I didn't realise the meta flags could be removed (I'm presuming just remove them from the config), that's great.

Basically the issue is that if they have meta flags when placed and broken by players in GM 1 they bypass all creative control plugins and drop the block. While they can't pick it up in creative mode, switching to survival allows them to obtain a 'free' precious stone, plus any other players they wish to abuse it for.

 @frizzbee30
  
frizzbee30 commented on May 29, 2016
That seems to have done the trick :) wish I'd realised that a while ago, would have saved a lot of testing.. Thanks for the quick feedback, and the continued updates to a great plugin! :D

**Comment by ronc617** [comment]
Yes it seems that if I have the meta-autoset: to false
It does not register the placing of the field.
